src/sentry/api/endpoints/project_rule_details.py

```diff
@@ -49,11 +45,14 @@ def get(self, request: Request, project, rule) -> Response:
         for action in serialized_rule.get("actions", []):
             if action.get("_sentry_app_installation") and action.get("_sentry_app_component"):
                 installation = SentryAppInstallation(**action.get("_sentry_app_installation", {}))
-                component = installation.prepare_ui_component(
-                    SentryAppComponent(**action.get("_sentry_app_component")),
-                    project,
+                component = SentryAppComponent(**action.get("_sentry_app_component"))
```

Comment:
Very small rearrangement of code, in the future this may move into a service method ( wasn't necessary yet )